src/main/java/it/pagopa/interop/probing/eservice/registry/updater/service/EserviceService.java

```diff
@@ -0,0 +1,81 @@
+/**************************************************************************
+*
+* Copyright 2023 (C) DXC
+*
+* Created on  : 6 Mar 2023
+* Author      : dxc technology
+* Project Name: interop-be-probing-eservice-registry-updater 
+* Package     : it.pagopa.interop.probing.eservice.registry.updater.service
+* File Name   : EserviceService.java
+*
+*-----------------------------------------------------------------------------
+* Revision History (Release )
+*-----------------------------------------------------------------------------
+* VERSION     DESCRIPTION OF CHANGE
+*-----------------------------------------------------------------------------
+** --/1.0  |  Initial Create.
+**---------|------------------------------------------------------------------
+***************************************************************************/
+package it.pagopa.interop.probing.eservice.registry.updater.service;
+
+import java.io.IOException;
+import java.util.Objects;
+import java.util.UUID;
+
+import javax.transaction.Transactional;
+
+import it.pagopa.interop.probing.eservice.registry.updater.dao.EserviceDao;
+import it.pagopa.interop.probing.eservice.registry.updater.dto.EserviceDTO;
+import it.pagopa.interop.probing.eservice.registry.updater.model.Eservice;
+import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceState;
+import it.pagopa.interop.probing.eservice.registry.updater.util.EserviceTechnology;
+
+/**
+ * The Class EserviceServiceImpl.
+ */
+@Transactional
+public class EserviceService {
```

Comment:
Use Interface

Reply:
the singleton and the factory(service/serviceImpl) are two different creation patterns.
In our case the factory is not needed, as the behavior of the service will always be the same and we do not do dependency injection we will also overlay the expected design pattern